packages/react-composites/src/composites/CallWithChatComposite/SidePane.tsx

```diff
@@ -89,29 +97,50 @@ export const EmbeddedPeoplePane = (props: {
   onPeopleButtonClick: () => void;
   mobileView?: boolean;
 }): JSX.Element => {
-  const { callAdapter, chatAdapter, inviteLink } = props;
+  const { callAdapter, chatAdapter, inviteLink, mobileView } = props;
   const participantListDefaultProps = usePropsFor(ParticipantList);
 
-  const callWithChatStrings = useCallWithChatCompositeStrings();
+  const locale = useLocale();
+
+  const [menuItems, setMenuItems] = useState<_DrawerMenuItemProps[]>([]);
 
-  const participantListProps = useMemo(() => {
+  const participantListProps: ParticipantListProps = useMemo(() => {
     const onRemoveParticipant = async (participantId: string): Promise<void> =>
       removeParticipantFromCallWithChat(callAdapter, chatAdapter, participantId);
     return {
       ...participantListDefaultProps,
-      onRemoveParticipant
+      onRemoveParticipant: mobileView ? undefined : onRemoveParticipant
```

Comment:
Add a comment here why is the onRemoveParticipant callback is not supplied in mobile view